src/components/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+
+import style from './Dashboard.module.scss';
+
+const activities = [
+  { activity: 'running', id: 1, timesDone: 3 },
+  { activity: 'swimming', id: 2, timesDone: 2 },
+  { activity: 'not smoking', id: 3, timesDone: 21 },
+];
+
+export const Dashboard = () => {
+  const [formVisible, setFormVisible] = useState<boolean>(false);
+  const [newActivity, setNewActivity] = useState<string>('');
+  const [currentActivities, setCurrentActivities] = useState([...activities]);
+
+  const handleChange = () => {
+    setNewActivity(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    event.preventDefault();
+    const newObj = { activity: newActivity, id: 4, timesDone: 0 };
+    const newArray = activities.concat([newObj]);
+    setCurrentActivities(newArray);
+    setFormVisible(false);
+  };
```

Comment:
Lets start adding some nice new lines between blocks of code

```suggestion
  const handleSubmit = () => {
    event.preventDefault();
    
    const newObj = { activity: newActivity, id: 4, timesDone: 0 };
    const newArray = activities.concat([newObj]);
    
    setCurrentActivities(newArray);
    setFormVisible(false);
  };
```
It is highly opinionated. The argument here could be if the first setter is part of the middle block or part of the block of setters. It is really up to you. But newline here and there helps with the readability and ease the cognitive load of the code

Reply:
For sure will follow ✅